src/diffusers/models/unet_sde_score_estimation.py

```diff
@@ -228,13 +229,14 @@ def __init__(
         self,
         image_size=1024,
         num_channels=3,
+        centered=False,
         attn_resolutions=(16,),
         ch_mult=(1, 2, 4, 8, 16, 32, 32, 32),
         conditional=True,
         conv_size=3,
         dropout=0.0,
         embedding_type="fourier",
-        fir=True,  # TODO (patil-suraj) remove this option from here and pre-trained model configs
+        fir=True,
```

Comment:
@patil-suraj - I don't think all models use the fir filter

Reply:
See messages on Slack

Reply:
Yes, this PR handles both `fir=True` and `fir=False` cases.